### ISSUE TITLE
Can not fixed navbar

### ISSUE THREAD

**Issue by ngoncom** [vmitsaras/js-offcanvas#32]
Hi,

I try to fixed navbar with css but not luck, the problem is translate(3d), do you have any jsfiddle

Thank!

**Comment by vmitsaras** [comment]
Hi,
you have to use the events (open, close) 
to manipulate the navbar. 
Is it Bootstrap?

**Comment by ngoncom** [comment]
Yes, I am using Bootstrap and I will try using js, thank for your reply.

**Comment by vmitsaras** [comment]
Codepen Demo?

**Comment by kevinmamaqi** [comment]
I have the same problem, when the `.sticky-top` bootstrap 4 class is used - https://getbootstrap.com/docs/4.1/utilities/position/ - it doesn't show because the `.c-offcanvas-content-wrap` has `overflow: hidden;`.

Is there a workaround for it?

**Comment by vmitsaras** [comment]
@kevinmamaqi 
i've already fixed that
https://github.com/vmitsaras/js-offcanvas/commit/1c95921f36e237ada2542c3bc295b9bf4351a78c#diff-2fd09665f1048906a2cb876145e62e66L23 

Can you update to the newest version and check again.
Thanks

**Comment by kevinmamaqi** [comment]
Thank you @vmitsaras it works now.

**Comment by vmitsaras** [comment]
thanks @kevinmamaqi
